<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasks handler</string>
  </property>
  <widget class="QGraphicsView" name="graphEditor"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNode"/>
   <addaction name="actionEdge"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/node.png</normaloff>:/images/node.png</iconset>
   </property>
   <property name="text">
    <string>Node</string>
   </property>
   <property name="toolTip">
    <string>Insert nodes</string>
   </property>
  </action>
  <action name="actionEdge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/edge.png</normaloff>:/images/edge.png</iconset>
   </property>
   <property name="text">
    <string>Edge</string>
   </property>
   <property name="toolTip">
    <string>Connect nodes with edge</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start calculation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
